src/Entity/Menu.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+final class Menu
+{
+    private $ref;
+
+    private $prix;
+
+    private $boisson;
+
+    private $plat;
+
+    private $accompagnement;
+
+    public function __construct()
+    {
+        $this->ref = \App\ORM\Util\UUID::v4();
+        $this->prix = 10.0;
```

Comment:
😨 😱 this is kinda arbitrary, it should be empty by default, and raise an error if not defined when you try to serialize it :) (see the `__sleep` and` __wakeup` methods)